api/src/uvisor-lib.c

```diff
@@ -19,21 +19,14 @@
 
 int uvisor_lib_init(void)
 {
-    /* Supported */
-    /* Defined in uvisor-input.S */
-    extern uint32_t uvisor_config;
-    extern uint32_t uvisor_export_table_size;
+    TUvisorExportTable * export_table = uvisor_export_table();
```

Comment:
@Patater use of const as we only read from the table? Consider returning a const from uvisor_export_table?